test/e2e_node/cpu_manager_test.go

```diff
@@ -422,7 +422,7 @@ func runCPUManagerTests(f *framework.Framework) {
 		framework.ExpectNoError(err, "expected log not found in container [%s] of pod [%s]",
 			pod.Spec.Containers[0].Name, pod.Name)
 
-		err = f.PodClient().MatchContainerOutput(pod.Name, pod.Spec.Containers[0].Name, expAllowedCPUsListRegex)
+		err = f.PodClient().MatchContainerOutput(pod.Name, pod.Spec.Containers[1].Name, expAllowedCPUsListRegex)
 		framework.ExpectNoError(err, "expected log not found in container [%s] of pod [%s]",
 			pod.Spec.Containers[1].Name, pod.Name)
```

Comment:
Yes, this index(`Containers[1]`) seems for comparing pod.Spec.Containers[1].Name and expAllowedCPUsListRegex.